modules/identity-provider-oidc-CR.adoc

```diff
@@ -84,15 +77,17 @@ spec:
         email: <6>
         - custom_email_claim
         - email
+        groups: <7>
```

Comment:
@stlaz Is at the end an okay place to put this? I checked the order of the claims in one of your PRs [1] and it was different, so I wasn't sure if this should all be rearranged to be correct.

[1] https://github.com/openshift/api/pull/1022/files#diff-8c86e336a0f7bb5d88279c3031020737f85b635494b094560f28e4301106a21dR299

Reply:
the order should not matter